libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -294,9 +283,13 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
      * Add FormField to FormGroup
      */
     private addToFormGroup(): void {
-        if (!this.formGroupContainer) {
+        // prevent adding wrapped fields into field group into form group container
+        const isFieldGroup = this._elementRef.nativeElement.parentElement.tagName.toLowerCase() === 'fdp-form-field-group';
```

Comment:
When we are using this long dotted path, can some of this be NULL? 
This might not with metaUI (the rule engine)

where we wrap these things with extra tag so it coudl like like:
```
<from-group>

<meta-ui-tag-that generantes bellow code>
     <form-group-field>

<meta-ui-tag-that generantes bellow code>

           ` <form-field>
 ```

So I would try to lookup closest element instead direct parent